docs/user-guide/multibroker.md

```diff
@@ -0,0 +1,46 @@
+# MultiBroker
+
+Starting in HELICS 2.5 there is a MultiBroker type that allows connection with multiple communication types simultaneously.  The multibroker allows an unlimited number of communication operations to interact.  
```

Comment:
There's trailing spaces in this sentence. Also, these sentences are separated by two spaces when others in the document are separated by one.